meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -26,9 +26,11 @@ SCRIPTS_VERSION ?= "1"
 BSP_VERSION ?= "0"
 PATCH_VERSION ?= "0"
 
-SDK_VERSION := "${@'${DISTRO_VERSION}'.replace('+snapshot-${DATE}','-snapshot')}"
-DISTRO_VERSION[vardepsexclude] = "DATE"
-SDK_VERSION[vardepsexclude] = "DATE"
+SDK_VERSION = "${@d.getVar('DISTRO_VERSION').replace('-alpha', '')}.${DATETIME_SECS}"
+SDK_VERSION[vardepsexclude] += "DATETIME_SECS"
+
+DATETIME_SECS = "${@int(time.mktime(time.strptime('${DATETIME}', '%Y%m%d%H%M%S')))}"
+DATETIME_SECS[doc] = "The timestamp from the DATETIME variable, as seconds since 1970."
```

Comment:
Hmmm isn't seconds way too granular? How about we use minutes?

Reply:
We certainly could, what's the concern with it? I'd like to note that we exclude this from the checksum, so while the timestamp will be included *when* the sdk task is run, the timestamp changing alone won't cause it to rebuild. That is, re-running your build won't build it from scratch just due to the hcange in the date, but if something else changes, it'll include the new timestamp.

Reply:
The original intention was to avoid conflicts if we rebuild, so just chose an arbitrary increasing number of the timestamp, but perhaps we should make it something actually user-comprehensible now that this isn't so specific to the ADE. I.e. YYYY.MM.DD.HH.MM or something?

Reply:
I'd go with the above suggestion that you provided i.e. YYYY.MM.DD.HH.MM.

> We certainly could, what's the concern with it?

Nothing in particular I'd say. Just the seconds looked a bit too granular as I said. I don't think SDKs would be generated with such a short diff in time but minutes looked plausible.

Reply:
The more user-friendly timestamp is actually longer than the other one, but potentially less confusing. Updating.